np-devops-team1/DevOps_Main
# Feat/start new game

## Description

This feature consists of functions that starts a new game with handling of game functions for SimpCity

## Changes

**Features**:
- Print Game Menu and Score
- Handles building count 
- Handles plotting of buildings

## Screenshots

**BEFORE**:
![image](https://user-images.githubusercontent.com/52743433/148010434-9781081a-d2a0-48ca-8b1e-62945ffcdd8a.png)

**AFTER**:
![image](https://user-images.githubusercontent.com/52743433/148010407-7dcc955a-ee16-462d-b68f-21b154aaddf2.png)

## Tests
Run start new game function 
- Run option 0 to 5 to test each function of the game 
- Run invalid option, expected outcome invalid option
